### ISSUE TITLE
[BUG] Fresh Install of 28.0.4-ls309 causes "Error: This app cannot be enabled because it makes the server unstable" after enabling apps. 

### ISSUE THREAD

**Issue by becksy82** [linuxserver/docker-nextcloud#422]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Enabling of apps causes Error: This app cannot be enabled because it makes the server unstable. I downloaded no apps. It happends with no automatically enabled apps like External Storage Support, and after dis and reenabling with activated apps.

Logs are not created for this error. Downgrading to 28.0.3-ls308 solved the problem.

### Steps To Reproduce

Happends after every fresh installation. And updated from 28.0.3-ls308 to 28.0.4-ls309

### Environment

```markdown
- OS: Unraid 6.12.9
- How docker service was installed: Unraid CA Store, so special configuartion
```

### CPU architecture

x86-64

### Docker creation

```bash
Standard Templ
```

### Container logs

```bash
root@unraid:~# docker logs nextcloud
[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] 02-default-location: skipped
[migrations] done
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝

   Brought to you by linuxserver.io
───────────────────────────────────────

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    99
User GID:    100
───────────────────────────────────────

using keys found in /config/keys
Initializing nextcloud 28.0.4.1 (this can take a while) ...
Upgrading nextcloud from 28.0.3.2 ...
Setting permissions
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Setting log level to debug
Turned on maintenance mode
Updating database schema
Updated database
Updating <circles> ...
Updated <circles> to 28.0.0
Updating <support> ...
Updated <support> to 1.11.1
Starting code integrity check...
Finished code integrity check
Update successful
Turned off maintenance mode
Resetting log level
Initializing finished
System config value memcache.local set to string \OC\Memcache\APCu
System config value filelocking.enabled set to boolean true
System config value memcache.locking set to string \OC\Memcache\APCu
System config value upgrade.disable-web set to boolean true
[custom-init] No custom files found, skipping...
[ls.io-init] done.
```

**Comment by aptalca** [comment]
Looks like upstream issue: https://github.com/nextcloud/server/issues/44685

**Comment by homerr** [comment]
Bit baffled by this to be honest - can't work out if you have or have not tried to install anything, and if you did, is it a tested app.

Seeing we have no other reports of this issue and I cannot reproduce it myself, I am going to close this off and invite you to carry this on Discord if you are still having issues as there are more unraid users around.

**Comment by homerr** [comment]
Also - see this on nextcloud repo. Could be related.

**Comment by maximweb** [comment]
Today, I updated to the latest table docker image and encountered the same error message when trying to enable an app as administrator on the nextcloud GUI.

> Also - see this on nextcloud repo. Could be related. [nextcloud/server#44685](https://github.com/nextcloud/server/issues/44685)

In my case, it was no redirect from https to http (as they discuss in the quoted issue), but my port was lost. I run the nextcloud externally using port 444 (because 443 is already occupied).

When the error appears, the browser console shows:
```
Refused to connect to 'https://<redacted ip>/apps/files/' because it violates the following Content Security Policy directive: "connect-src 'self'".
```
whereas it should be
```
https://<redacted ip>:444/apps/files/
```

Applying the fix in PR #435 seems to work, but makes the the urls ugly.

**Background**

A long time ago, in order to make the 444 port work, I had to adjust the default.config at two locations (did this ages ago):
[first](https://github.com/linuxserver/docker-nextcloud/blob/master/root/defaults/nginx/site-confs/default.conf.sample#L93-L94)
```
        location = /.well-known/carddav { return 301 https://$host:444/remote.php/dav/; }
        location = /.well-known/caldav  { return 301 https://$host:444/remote.php/dav/; }
```

[second](https://github.com/linuxserver/docker-nextcloud/blob/master/root/defaults/nginx/site-confs/default.conf.sample#L101)
```
        return 301 https://$host:444/index.php$request_uri;
```

I basicalle prepended `https://$host:444` on the `return 301` redirects.

Maybe other redirects / rewrites in the nginx config need similar treatment? Not sure which, though :/
